releng/org.eclipse.xtext.tycho.parent/pom.xml

```diff
@@ -83,18 +78,14 @@
 			</plugin>
 			<plugin>
 				<groupId>org.eclipse.tycho</groupId>
-				<artifactId>tycho-versions-plugin</artifactId>
-				<version>${tycho-version}</version>
-			</plugin>
-			<plugin>
-				<groupId>org.eclipse.tycho</groupId>
 				<artifactId>target-platform-configuration</artifactId>
 				<version>${tycho-version}</version>
 				<configuration>
 					<target>
 						<artifact>
 							<groupId>org.eclipse.xtext</groupId>
-							<artifactId>${target-platform-artifactId}</artifactId>
+							<artifactId>org.eclipse.xtext.target</artifactId>
+							<classifier>org.eclipse.xtext.target${target-platform-classifier}</classifier>
```

Comment:
no separator?

Reply:
Right, otherwise the default case does not work. The separator is part of the classifier.